scripts/run_backend_tests.py

```diff
@@ -337,21 +338,17 @@ def main(args=None):
     for directory in DIRS_TO_ADD_TO_SYS_PATH:
         if not os.path.exists(os.path.dirname(directory)):
             raise Exception('Directory %s does not exist.' % directory)
-        sys.path.insert(0, directory)
+        sys.path.insert(1, directory)
```

Comment:
Motivation for changing this: https://stackoverflow.com/a/10095099 and https://stackoverflow.com/q/10095037

Reply:
Please add a comment in the documentation, this seems important.

Do we need to fix it elsewhere?